wof/examples/flask/odm2/timeseries/runserver_odm2_timeseries.py

```diff
@@ -2,9 +2,6 @@
 
 import wof.flask
 
-curr_folder = os.path.abspath(os.path.dirname(__file__))
-sys.path.insert(0, '%s/../../../' % curr_folder)
```

Comment:
In https://github.com/ODM2/WOFpy/pull/76/commits/f30879b30c680ba64c3aea7df8f318fb976bd7f8 I fixed this to work in any platform but I don't think this is needed at all. All it does is to add the example folder to the path and, if the user is expected to run from that folder, this is not needed. Please let me know if I am getting this wrong.